broker-cli/commands/order/summary.js

```diff
@@ -49,12 +49,17 @@ function createUI (market, orders) {
  * @param {Logger} logger
  */
 async function summary (args, opts, logger) {
-  const { market, rpcAddress } = opts
+  const { market, rpcAddress, json } = opts
   const request = { market }
   try {
     const brokerDaemonClient = new BrokerDaemonClient(rpcAddress)
     const orders = await brokerDaemonClient.orderService.getBlockOrders(request)
-    createUI(market, orders.blockOrders)
+
+    if (json) {
+      console.log(JSON.stringify(orders))
```

Comment:
from a consistency standpoint, in most of the other commands you do an `if...return` but here you do an `if...else`. any reason for the difference? If not, I would say settle on one style and stick with it.